packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/doenet/boolean/test.rs

```diff
@@ -0,0 +1,62 @@
+use super::*;
+
+use test_helpers::*;
+
+use crate::{
+    components::prelude::StateVarIdx, test_utils::dast_root_no_position, ComponentIdx, DoenetMLCore,
+};
+
+#[test]
+fn boolean_sv_is_alias_of_value() {
+    let dast_root = dast_root_no_position(r#"<boolean>true</boolean>"#);
+    let mut core = DoenetMLCore::new(dast_root, "", "", None);
+
+    // the boolean will be index 1, as the document tag will be index 0.
+    let boolean_idx = 1;
+
+    assert_eq!(get_value_prop(boolean_idx, &mut core), true);
+    assert_eq!(get_boolean_prop(boolean_idx, &mut core), true);
```

Comment:
The name of this test seems to suggest that you're asserting their equal to each other, not to true necessarily.

Reply:
fair enough